src/main/java/frc/robot/RobotContainer.java

```diff
@@ -206,7 +207,18 @@ private void configureBindings() {
         .rightBumper()
         .onTrue(Sequences.elevatorFarHighFromAmp(m_elevator, m_shooter, m_led));
 
-    m_coDriverController.b().onTrue(Sequences.shoot(m_shooter, m_elevator, m_led));
+    m_coDriverController
+        .b()
+        .onTrue(Sequences.shoot(m_shooter, m_elevator, m_led))
+        .onFalse(
+            Commands.runOnce(
+                () -> {
+                  Double m_shootingTimeOnFalse = Timer.getFPGATimestamp();
```

Comment:
Will always be > 3

Reply:
Devrait avoir une boucle qui regarde le timestamp actuel - le timestamp de d/but de la boucle

Reply:
Use WaitUntilCommand(() -> (Timer.getFPGATimestamp() - m_shootingTimeOnFalse) > 3;

Reply:
"""
runOnce( () -> m_shootingTimeOnFalse = Timer.getFPGATimestamp() )
WaitUntilCommand ( .... )
finally ( stop aand reset timeer)

"""